.github/workflows/cd.yml

```diff
@@ -9,21 +17,27 @@ env:
   VERSION: "1.0.0"
 
 jobs:
-  ci:
```

Comment:
Not running CI feels *very* unsafe to me, the whole goal of CI is to make sure that if the `main` is green - everything is fine, it doesn't matter what and when is tested in other branches, PRs, etc. Now we give up this safety for optimization in a repo where anyone can merge as administrator, with a growing team that will fail us sooner than later.

Reply:
Updated to run CI in CD

Reply:
So I now realize the issue with this -- we will never be able to publish breaking changes since `compatibility_tests` will always be red on main.

I think the workaround could be to disable compatibility testing on `main` and only run on PRs and hotfixes.